iothub/device/src/Microsoft.Azure.Devices.Client.csproj

```diff
@@ -22,7 +22,7 @@
   </PropertyGroup>
 
   <PropertyGroup>
-    <Version>1.19.0</Version>
+    <Version>1.19.1</Version>
```

Comment:
Agree about the version update. We should only update the versions.csv file, and leave the csproj to be updated during release